thirdeye/thirdeye-frontend/app/utils/manage-alert-utils.js

```diff
@@ -287,6 +287,24 @@ export function buildAnomalyStats(alertEvalMetrics, anomalyStats, showProjected
   return anomalyStats;
 }
 
+/**
+ * Returns selected alert object properties for config group table display
+ * @param {Object} alertData - a single alert record
+ * @param {Number} cnt - record index
```

Comment:
Can we rename `cnt` to something more meaningful?

Reply:
`index`or `alertIndex` perhaps?

Reply:
`alertIndex`